### DISCUSSION TITLE
how to allow bot which responds on 1 to 1 to respond in groups?

### DISCUSSION THREAD

**Discussion by giorgiberia** [python-telegram-bot/python-telegram-bot#2639] [Q&A] [answered]
the bot responds to me on 1 on 1  according to different commands that I defined.
but when i add bot to group it does not respond there to anyone

**Comment by Welsyntoffie** [comment, marked as answer]
> the bot responds to me on 1 on 1 according to different commands that I defined.
> but when i add bot to group it does not respond there to anyone

Try giving the bot admin rights. It worked for me.

**Reply by Bibo-Joshi** [reply]
IIRC this essentially deactivates privacy mode for that group, which basically boils down to harshils answers :)

**Reply by Welsyntoffie** [reply]
but its still acceptable right?

**Reply by Bibo-Joshi** [reply]
depends on the use case - as does disableling privacy mode. just wanted to provide some context :)

**Comment by harshil21** [comment]
See https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#what-messages-can-my-bot-see
